code/modules/mapping/reader.dm

```diff
@@ -304,6 +306,10 @@
 	//Instanciation
 	////////////////
 
+	for (var/turf/turf_in_blacklist in turf_blacklist)
+		if (crds == turf_in_blacklist) //if the given turf is blacklisted, dont do anything with it
```

Comment:
```suggestion
	for (var/t in turf_blacklist)
		var/turf/turf_in_blacklist = t
		if (crds == turf_in_blacklist) //if the given turf is blacklisted, dont do anything with it
```

Reply:
originally it was like that, i changed it to what it is now because anturk requested it

Reply:
They said there was no need to type, not to type-check.
Let me put a different suggestion to comply with what they wanted.
```suggestion
	for (var/turf_in_blacklist in turf_blacklist)
		if (crds == turf_in_blacklist) //if the given turf is blacklisted, dont do anything with it
```